src/record.rs

```diff
@@ -106,15 +152,19 @@ impl Record {
         &self.header
     }
 
-    /// # Description
-    /// Serialize record to bytes
-    pub fn to_raw(&self) -> bincode::Result<Vec<u8>> {
+    pub(crate) fn to_partially_serialized(&self) -> Result<PartiallySerializedRecord> {
```

Comment:
This logic should be slightly modified after https://github.com/qoollo/pearl/pull/212 will be merged